Rakefile

```diff
@@ -99,7 +99,7 @@ namespace :crowbar do
     end
   end
 
-  desc "Add upstream remote to the crowbar organisation"
+  desc "Add upstream remote to git:crowbar/crowbar"
```

Comment:
s/to/pointing to/ or s/to/for/, since currently it sounds a bit like the upstream remote is being added to a remote repo, which obviously doesn't make sense.